### PR TITLE
Release 2.0.2

### PR DESCRIPTION
**Pull request by pankajastro**

### CODE REVIEW

File: CHANGELOG.md
Comment (Lee-W) [inline CHANGELOG.md:3]: Is there any blocker for this release? Or should we change `TBD` to today?

Comment (pankajastro) [reply CHANGELOG.md:3]: No blocker, waiting for a approval then I'll update the date

**Review by Lee-W — APPROVED** [review]

**Review by phanikumv — APPROVED** [review]

**Pull request merged**
